src/main/java/com/mongodb/shardsync/command/sync/SyncMetadataCommand.java

```diff
@@ -0,0 +1,55 @@
+package com.mongodb.shardsync.command.sync;
+
+import com.mongodb.shardsync.ShardConfigSync;
+import com.mongodb.shardsync.SyncConfiguration;
+
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+import java.util.concurrent.Callable;
+
+@Command(name = "metadata", description = "Synchronize shard metadata between clusters")
+public class SyncMetadataCommand implements Callable<Integer> {
+    
+    @CommandLine.ParentCommand
+    private SyncCommand parent;
+    
+    @Option(names = {"--optimized"}, 
+            description = "Combine adjacent chunks for optimization (default: true)", 
+            defaultValue = "true")
+    private boolean optimized = true;
+    
+    @Option(names = {"--legacy"}, 
+            description = "Use legacy metadata sync method (slower)", 
+            hidden = true)
+    private boolean legacy;
+    
+    @Option(names = {"--skipFlushRouterConfig"}, 
+            description = "Skip the flushRouterConfig step")
+    private boolean skipFlushRouterConfig;
+    
+    @Option(names = {"--force"}, 
```

Comment:
```suggestion
    @Option(names = {"--force"}, 
```
Best IMO would be to have the “preflight” be separate functionality.
Failing that, though, ISTM `--ignorePreflightCheckFailures` is improves on the experience versus simply skipping the checks.

Reply:
I'm somewhat reluctant to make the preflight optional/separate. The reason being is that typically when there are issues due to operator error, it's because a step was missed. A pretty common sequence is something like this:
1. Operator does sync (following prescribed steps)
2. Something blows up or fails in some way, prompting the operator to restart the migration
3. Operator does not properly follow the "reset" procedure, skipping entirely or missing some steps
4. 2nd attempt migration proceeds with lingering data/initialization from the first attempt

I will change the option to `ignorePreflightCheckFailures`